forgerock-auth/src/main/java/org/forgerock/android/auth/webauthn/FRWebAuthn.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2023 ForgeRock. All rights reserved.
+ *
+ *  This software may be modified and distributed under the terms
+ *  of the MIT license. See the LICENSE file for details.
+ */
+
+package org.forgerock.android.auth.webauthn
+
+import android.content.Context
+import org.forgerock.android.auth.WebAuthnDataRepository
+
+/**
+ * Manage [PublicKeyCredentialSource] that generated by the SDK.
+ * The [PublicKeyCredentialSource] only contains the reference to the actual key,
+ * deleting the [PublicKeyCredentialSource] does not delete the actual key
+ */
+class FRWebAuthn(private val context: Context,
+                 private val webAuthnDataRepository: WebAuthnDataRepository =
+                     WebAuthnDataRepository.WebAuthnDataRepositoryBuilder().context(context)
+                         .build()) {
+
+    /**
+     * Delete the [PublicKeyCredentialSource] by Relying Party Id
```

Comment:
can we also add the parameter (rpId) in the Doc ,

Reply:
Updated